package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "@prisma/client": "^5.12.0",
+    "next": "^14.1.4"
+  }
+}
```

Comment:
Be careful about using yarn outside of the `frontend/` folder! There should never be `package.json` and `yarn.lock` files outside of `frontend/`.